packages/manager/apps/web/client/app/domain/domain.constant.js

```diff
@@ -2,6 +2,13 @@ export const ALERTS = {
   tabs: 'domain_alert_tabs',
 };
 
+export const HOSTING_ORDERED_OFFER = [
+  'start1m',
+  'start10m',
+  'START_1_M',
```

Comment:
Can we `snakeCase` + `upperCase` here ? instead of duplicating reference ? (Also could those be get from an enum or something because I feel like we shouldn't have to maintain those

Reply:
I can retrieve all the Hosting web offers, but not just these specific ones.